runtime/domain.c

```diff
@@ -630,28 +699,146 @@ CAMLexport void caml_reset_domain_lock(void)
   return;
 }
 
-void caml_init_domains(uintnat minor_heap_wsz) {
-  int i;
-  uintnat size;
-  void* heaps_base;
+/* minor heap initialization and resizing */
 
+static uintnat get_minor_heap_reservation_size() {
```

Comment:
```suggestion
Caml_inline uintnat get_minor_heap_reservation_size() {
```

Reply:
Also the two places where `get_minor_heap_reservation_size` is used, the result is immediately passed to `reserve_minor_heaps`. Best to push `get_minor_heap_reservation_size` code into `reserve_minor_heaps` and delete that function.